introduce more additionals (#351)
the-teachingRSE-project/competencies: competencies.md

While this community paper is based on workshop discussions that were attended
largely by RSEs (deRSE23 in Paderborn, un-deRSE23 in
Jena, and deRSE24 in Würzburg, all in Germany),

> we believe that the competencies formulated here, centred around data handling, can offer far-reaching

Reviewer: I'm not actually sure about the "centered around data [ processing | handling ]", this has been added since the version I reviewed over the last couple of days and I think it changes the meaning somewhat. I might be missing the context here but the original text read:

> While this community paper is based on workshop discussions that were attended largely by Research Software Engineers (deRSE23 in Paderborn, un-deRSE23 in Jena, Germany), we believe that the competencies formulated here can offer far-reaching impact beyond the domain of RSE into adjacent fields of science...

This effectively says that while the majority of the people who participated in the discussions leading to this paper were RSEs we recognise that the competencies developed here have a much wider potential for impact across areas including data (and perhaps research computing infrastructure?).

As I understand it, the addition of "_...centred around data processing..._" has changed the meaning to say that the competencenies we have formulated here, while developed by a group largely made up of RSEs, are actually centred around data - I don't agree with that, they cover a whole range of areas across software, research and communication - while it's only a few additional words, it's quite early on in the core introductory material and I think it changes the focus of the paper.

Apologies if I'm misunderstanding, but if I am understanding this correctly, I'd suggest reverting to the original text:
```suggestion
we believe that the competencies formulated here can offer far-reaching
```

-->

#### \${DOMAIN}-RSE
While software is the common focus of all RSEs,
there will be RSEs that have additionally specialised in the intricacies of one particular research domain,
such as medical RSEs, digital humanities RSEs, or physics RSEs.
This can often serve as a base domain for RSE specialisation as in @subsec:examplemaster.

#### Data-focused RSE

> These are RSEs working at the flourishing intersection between data science and RSE.

Reviewer: I guess this is OK but when we say "These are", when reading it, it feels a little "impersonal". I might be inclined to say:
```suggestion
Data-focused RSEs work at the flourishing intersection between data science and RSE.
```
